frontend/packages/gitops-plugin/src/components/details/GitOpsResourcesSection.scss

```diff
@@ -0,0 +1,12 @@
+.odc-gitops-resources {
```

Comment:
As mentioned in our chat, we'll want to convert `odc` prefixes to something for gitops; `gop` (GitOps Plugin)

Reply:
Added to issue [GITOPS-1277](https://issues.redhat.com/browse/GITOPS-1277)